src/collections/programs/sca-contributhon/index.mdx

```diff
@@ -180,7 +180,6 @@ May 8, 2021 to May 14, 2021 - SCA announces successful participants and awards $
 <br />
 <b>Mentors</b>
```

Comment:
Remove this section completely

Reply:
Mentors section

Reply:
Ok thank you for telling.

Reply:
You removed the mentors section at the wrong line.
You need to remove it here at line: 176

Reply:
Oh I am sorry, I will close the PR and try again.
Thank you.